ui/main/src/app/modules/cards/components/card/card.component.ts

```diff
@@ -55,7 +86,113 @@ export class CardComponent implements OnInit, OnDestroy {
                     this.currentPath = urlParts[1];
                 }
             });
-        this.computeDisplayedDate();
+
+        this.userService.currentUserWithPerimeters()
+            .pipe(takeUntil(this.ngUnsubscribe))
+            .subscribe(userWithPerimeters => {
+                if (userWithPerimeters) {
+                    this.userWithPerimeters = userWithPerimeters;
```

Comment:
Why using this global variable userWithPerimeters as it is anly used in this method ?

Reply:
OK